pexpect/pxssh.py

```diff
@@ -95,7 +95,9 @@ class pxssh (spawn):
 
     def __init__ (self, timeout=30, maxread=2000, searchwindowsize=None,
                     logfile=None, cwd=None, env=None, ignore_sighup=True, echo=True,
-                    options={}, encoding=None, codec_errors='strict'):
+                    options={}, encoding=None, codec_errors='strict',
+                       password_regex='(?i)(?:password:)|(?:passphrase for key)',
+                       disable_original_sync=False):
```

Comment:
Style point: I prefer to avoid negative boolean parameters like `disable_foo`, unless there's a clear reason that the negative makes sense.

Looking at the rest of this class, I think the parameter fits better in the `login()` method (which already has parameters affecting syncing and changing the prompt). I'd probably just call it `sync_original_prompt`.